readthedocs/core/symlink.py

```diff
@@ -152,7 +146,9 @@ def symlink_cnames(self, domain=None):
         else:
             domains = Domain.objects.filter(project=self.project)
         for dom in domains:
-            self._log(u"Symlinking CNAME: {0} -> {1}".format(dom.domain, self.project.slug))
+            log_msg = u"Symlinking CNAME: {0} -> {1}".format(dom.domain, self.project.slug)
```

Comment:
Use single quotes `'` on literal strings, also we could get rid of the `u` by importing the `unicode_literals` or run `isort` to do it automatically :)

Travis is falling because of the linter, you can run the linter locally with`tox -e py27-lint`

Reply:
And please, could you don't touch the `readthedocs/doc_builder/environments.py` file? I'm taking care of the logging issue while refactoring on #3687, thanks!